src/views/ApplicationViews.js

```diff
@@ -19,46 +19,28 @@ import { NewTag } from "../components/tags/NewTag"
 // receiving 2 props from Rare.js
 // responsible for routing users to specific views depending on URL paths
 export const ApplicationViews = ({ token, setToken }) => {
-  return (
-    <>
-      <Routes>
-        {/* creating path for login & register */}
-        {/* passing setToken prop from Rare.js to Login & Register  */}
-        <Route path="/login" element={<Login setToken={setToken} />} />
-        <Route path="/register" element={<Register setToken={setToken} />} />
-        <Route path="/">
-          <Route index element={<SubscriptionList token={token} />} />
+  return <>
+    <Routes>
+      {/* creating path for login & register */}
```

Comment:
This is missing several lines from the current main branch. I would suggest going to current main, copying the entire module and replacing yours with the current.